Gemfile

```diff
@@ -112,6 +112,7 @@ gem 'zero_downtime_migrations'
 group :development do
   gem 'benchmark-ips'
   gem 'guard-rubocop'
+  gem 'httparty'
```

Comment:
Dislike, we already have `Faraday` and ruby has `Net::HTTP` , even though it's only in dev I don't see the need to add another external dependency

Reply:
Removed it.